README.md

```diff
@@ -317,3 +326,48 @@ SYCL and the SYCL logo are trademarks of the Khronos Group Inc.
 [lesson-10-source]: ./Code_Exercises/Exercise_10_Managing_Dependencies/source.cpp
 [lesson-10-solution]: ./Code_Exercises/Exercise_10_Managing_Dependencies/solution.cpp
 
+[lesson-11-slides]: ./Lesson_Materials/Lecture_11_In_Order_Queue/
+[lesson-11-exercise]: ./Code_Exercises/Exercise_11_In_Order_Queue/doc.md
+[lesson-11-source]: ./Code_Exercises/Exercise_11_In_Order_Queue/source.cpp
+[lesson-11-solution]: ./Code_Exercises/Exercise_11_In_Order_Queue/solution.cpp
+
+[lesson-12-slides]: ./Lesson_Materials/Lecture_12_Temporary_Data/
```

Comment:
I think we should keep this one as "Lecture_12_Advanced_Data_Flow" as it talks about a few data-flow related topics.

Reply:
Change made.